tests/integration/TechRecordsFunction.intTest.ts

```diff
@@ -487,19 +493,17 @@ describe("downloadDocument", () => {
 
           await LambdaTester(DownloadDocumentFunction)
             .event({
-              path: `/vehicles/${techRecord.vin}/download-file/someFilename.pdf`,
+              path: `/vehicles/${techRecord.vin}/download-file?filename=someFilename.pdf`,
```

Comment:
path and test description don't match. Can just remove the queryStringParam

Reply:
Still true. Still needs to change

Reply:
done